src/pal/src/include/pal/palinternal.h

```diff
@@ -325,6 +325,7 @@ function_name() to call the system's implementation
 #define FILE PAL_FILE
 
 #include "pal.h"
+#include "palprivate.h"
```

Comment:
I'm not sure why the functions that you've moved to the palprivate.h didn't actually go to palinternal.h.